.github/workflows/test-ci.yml

```diff
@@ -0,0 +1,23 @@
+name: test-ci
```

Comment:
the name test-ci is too generic, actually it's just unit test, we will have e2e tests in the future, so `unit-test.yaml` is better.

Reply:
ok, use unit-test-ci.yml